app/src/components/PreviousDonations/PreviousDonationsTable.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PageHoc from '../PageHoc/PageHoc';
+import Table from '../Table';
+import Button from '../Button/Button';
+
+const actionInfo = (name, buttonType, onClick, isFreeAction = undefined) =>
+  isFreeAction
+    ? { icon: 'none', name, buttonType, onClick, isFreeAction }
+    : { icon: 'none', name, buttonType, onClick };
+
+const columns = [
+  {
+    field: 'name',
+    sorting: false,
+    render: rowData => {
+      if (
+        rowData.contributorType === 'individual' ||
+        rowData.contributorType === 'family'
+      ) {
+        return `${rowData.firstName} ${rowData.lastName}`;
+      }
+      return rowData.name;
+    },
+  },
+  {
+    field: 'campaign',
+    render: rowData => {
+      return rowData && rowData.campaign ? rowData.campaign.name : 'Loading...';
+    },
+  },
+  {
+    field: 'amount',
+    type: 'currency',
+  },
+  {
+    field: 'date',
+    render: rowData =>
+      new Date(rowData.date)
+        .toLocaleString('en-US', {
+          year: 'numeric',
+          month: '2-digit',
+          day: '2-digit',
+        })
+        .split(', ')[0],
+  },
+];
+
+class PreviousDonationsTable extends React.Component {
+  render() {
+    const {
+      isListLoading,
```

Comment:
isListLoading may be undefined, could be removed.

Reply:
you're right